Tests/AblyChatTests/IntegrationTests.swift

```diff
@@ -82,25 +96,99 @@ struct IntegrationTests {
         let rxReactionFromSubscription = try #require(await rxReactionSubscription.first { _ in true })
         #expect(rxReactionFromSubscription.type == "heart")
 
+        // MARK: - Occupancy
+
+        // It can take a moment for the occupancy to update from the clients connecting above, so we’ll wait a 2 seconds here.
+        try await Task.sleep(nanoseconds: 2_000_000_000)
+
+        // (12) Get current occupancy
+        let currentOccupancy = try await rxRoom.occupancy.get()
+        #expect(currentOccupancy.connections != 0) // this flucuates dependant on the number of clients connected e.g. simulators running the test, hence why checking for non-zero
+        #expect(currentOccupancy.presenceMembers == 0) // not yet entered presence
+
+        // (13) Subscribe to occupancy
+        let rxOccupancySubscription = await rxRoom.occupancy.subscribe(bufferingPolicy: .unbounded)
+
+        // (14) Enter presence on the other client and check that we receive the updated occupancy on the subscription
+        try await txRoom.presence.enter(data: nil)
+
+        // It can take a moment for the occupancy to update from the clients entering presence above, so we’ll wait 2 seconds here.
+        try await Task.sleep(nanoseconds: 2_000_000_000)
+
+        // (15) Check that we received an updated presence count when getting the occupancy
+        let updatedCurrentOccupancy = try await rxRoom.occupancy.get()
+        #expect(updatedCurrentOccupancy.presenceMembers == 1) // 1 for txClient entering presence
```

Comment:
And here you check an exact number, I don't get it.

Reply:
this checks presenceMembers, the other was connections. we have finer control over presence (as you have to enter or leave it)